interfaces/Portal/src/app/pages/payment/payment.page.ts

```diff
@@ -14,7 +14,7 @@ export class PaymentPage {
   public Permission = Permission;
 
   public programId = this.route.snapshot.params.id;
-  public thisPhase = ProgramPhase.payment;
+  public thisPhase = ProgramTab.payment;
   public isReady: boolean;
```

Comment:
I think this whole "ready"-state thing per phase is irrelevant by now...
We might be able to "just delete it all" already...